components/support/migration/src/main/java/mozilla/components/support/migration/FennecMigrator.kt

```diff
@@ -549,6 +549,8 @@ class FennecMigrator private constructor(
 
         runBlocking {
             store.dispatch(MigrationAction.Started).join()
+
+            emitStartedFact()
```

Comment:
nit: Technically doesn't need to be in `runBlocking` here. Maybe move it after starting the service?